src/invoices/InvoicesList/InvoicesListSearchConfig.js

```diff
@@ -1,7 +1,7 @@
 const indexes = [
   'voucherNumber',
   'vendorInvoiceNo',
-  'poNumbers',
+  'poNumber',
```

Comment:
plz check filter itself, `poNumber` is used as search value, not only translation

Reply:
updated